2023-05-12 14:23:47,193 - Abs测试报告 - INFO - ----------base用例接口测试开始----------
2023-05-12 14:23:47,427 - Abs测试报告 - INFO - **********base用例接口测试结束**********
2023-05-12 14:23:47,428 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-12 14:23:47,429 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:23:47,429 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:23:47,594 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:23:47,594 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:23:47,707 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:23:47,708 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:23:47,842 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-12 14:26:10,020 - Abs测试报告 - INFO - ----------base用例接口测试开始----------
2023-05-12 14:26:10,259 - Abs测试报告 - INFO - **********base用例接口测试结束**********
2023-05-12 14:26:10,261 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-12 14:26:10,261 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:26:10,262 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:26:10,384 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:26:10,385 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:26:10,499 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:26:10,499 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:26:10,603 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-12 14:29:04,034 - Abs测试报告 - INFO - ----------base用例接口测试开始----------
2023-05-12 14:29:04,271 - Abs测试报告 - INFO - **********base用例接口测试结束**********
2023-05-12 14:29:04,273 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-12 14:29:04,273 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:29:04,274 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:29:04,398 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:29:04,398 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:29:04,503 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:29:04,503 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:29:04,625 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-12 14:35:34,982 - Abs测试报告 - INFO - ----------base用例接口测试开始----------
2023-05-12 14:35:35,210 - Abs测试报告 - INFO - **********base用例接口测试结束**********
2023-05-12 14:35:35,212 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-12 14:35:35,212 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:35:35,212 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:35:35,344 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:35:35,344 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:35:35,455 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:35:35,455 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:35:35,581 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - ==========start==========
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例加法开始测试
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例加法测试通过
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例减法开始测试
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例减法测试通过
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例乘法开始测试
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例乘法测试通过
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例除法开始测试
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - 用例除法测试通过
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - **********end**********
2023-05-12 14:36:06,464 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-12 14:36:06,585 - Abs测试报告 - INFO - 注册成功
2023-05-12 14:36:06,586 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-12 14:36:06,700 - Abs测试报告 - INFO - 登录成功
2023-05-12 14:36:06,700 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:36:06,700 - Abs测试报告 - DEBUG - url:http://api.lemonban.com/futureloan/member/recharge
2023-05-12 14:36:06,700 - Abs测试报告 - DEBUG - method:post
2023-05-12 14:36:06,700 - Abs测试报告 - DEBUG - request:{'headers': {'X-Lemonban-Media-Type': 'lemonban.v2', 'Authorization': 'BearereyJhbGciOiJIUzUxMiJ9.eyJtZW1iZXJfaWQiOjEwNzk1MSwiZXhwIjoxNjgzODczNjY1fQ.JNokxvUtx-5Pz-3e0OLTgmEXvf_h2O6UUyVQO_kzG3ZIdvmlavCIRLFGuYS8PrUaGpGXvT6ukrw7aQabhfRzIA'}, 'json': {'member_id': 107951, 'amount': 666}}
2023-05-12 14:36:06,762 - Abs测试报告 - ERROR - 请求结果断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\test_recharge.py", line 91, in test_recharge
    self.assertEqual(case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
- {'code': '0', 'msg': 'OK'}
+ {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,763 - Abs测试报告 - DEBUG - 期望数据:{'code': '0', 'msg': 'OK'}
2023-05-12 14:36:06,763 - Abs测试报告 - DEBUG - 实际结果:{'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,763 - Abs测试报告 - DEBUG - 响应结果:{'code': 1003, 'copyright': 'Copyright 柠檬班 © 2017-2020 湖南省零檬信息技术有限公司 All Rights Reserved', 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,763 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:36:06,763 - Abs测试报告 - DEBUG - url:http://api.lemonban.com/futureloan/member/recharge
2023-05-12 14:36:06,763 - Abs测试报告 - DEBUG - method:post
2023-05-12 14:36:06,763 - Abs测试报告 - DEBUG - request:{'headers': {'X-Lemonban-Media-Type': 'lemonban.v2', 'Authorization': 'BearereyJhbGciOiJIUzUxMiJ9.eyJtZW1iZXJfaWQiOjEwNzk1MSwiZXhwIjoxNjgzODczNjY1fQ.JNokxvUtx-5Pz-3e0OLTgmEXvf_h2O6UUyVQO_kzG3ZIdvmlavCIRLFGuYS8PrUaGpGXvT6ukrw7aQabhfRzIA'}, 'json': {'member_id': 107951, 'amount': 666.6}}
2023-05-12 14:36:06,835 - Abs测试报告 - ERROR - 请求结果断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\test_recharge.py", line 91, in test_recharge
    self.assertEqual(case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
- {'code': '0', 'msg': 'OK'}
+ {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,836 - Abs测试报告 - DEBUG - 期望数据:{'code': '0', 'msg': 'OK'}
2023-05-12 14:36:06,836 - Abs测试报告 - DEBUG - 实际结果:{'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,836 - Abs测试报告 - DEBUG - 响应结果:{'code': 1003, 'copyright': 'Copyright 柠檬班 © 2017-2020 湖南省零檬信息技术有限公司 All Rights Reserved', 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,836 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:36:06,836 - Abs测试报告 - DEBUG - url:http://api.lemonban.com/futureloan/member/recharge
2023-05-12 14:36:06,836 - Abs测试报告 - DEBUG - method:post
2023-05-12 14:36:06,836 - Abs测试报告 - DEBUG - request:{'headers': {'X-Lemonban-Media-Type': 'lemonban.v2', 'Authorization': 'BearereyJhbGciOiJIUzUxMiJ9.eyJtZW1iZXJfaWQiOjEwNzk1MSwiZXhwIjoxNjgzODczNjY1fQ.JNokxvUtx-5Pz-3e0OLTgmEXvf_h2O6UUyVQO_kzG3ZIdvmlavCIRLFGuYS8PrUaGpGXvT6ukrw7aQabhfRzIA'}, 'json': {'member_id': 107951, 'amount': 666.66}}
2023-05-12 14:36:06,903 - Abs测试报告 - ERROR - 请求结果断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\test_recharge.py", line 91, in test_recharge
    self.assertEqual(case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
- {'code': '0', 'msg': 'OK'}
+ {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,904 - Abs测试报告 - DEBUG - 期望数据:{'code': '0', 'msg': 'OK'}
2023-05-12 14:36:06,904 - Abs测试报告 - DEBUG - 实际结果:{'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,904 - Abs测试报告 - DEBUG - 响应结果:{'code': 1003, 'copyright': 'Copyright 柠檬班 © 2017-2020 湖南省零檬信息技术有限公司 All Rights Reserved', 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:36:06,904 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-12 14:36:06,904 - Abs测试报告 - INFO - ----------base用例接口测试开始----------
2023-05-12 14:36:06,905 - Abs测试报告 - INFO - **********base用例接口测试结束**********
2023-05-12 14:36:06,905 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-12 14:36:06,905 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:36:06,905 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:36:07,011 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:36:07,011 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:36:07,122 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:36:07,122 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:36:07,233 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-12 14:38:06,202 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-12 14:38:06,204 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:38:06,204 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-12 14:38:07,033 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:38:07,034 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-12 14:38:09,779 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:38:09,781 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-12 14:38:10,596 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-12 14:45:07,286 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-12 14:45:07,405 - Abs测试报告 - INFO - 注册成功
2023-05-12 14:45:07,405 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-12 14:45:07,517 - Abs测试报告 - INFO - 登录成功
2023-05-12 14:45:07,518 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:45:07,519 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:45:07,821 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:45:07,821 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:45:07,903 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:45:07,903 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:45:07,988 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-12 14:50:29,517 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-12 14:50:29,637 - Abs测试报告 - INFO - 注册成功
2023-05-12 14:50:29,638 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-12 14:50:29,753 - Abs测试报告 - INFO - 登录成功
2023-05-12 14:50:29,754 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:50:29,754 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:50:29,824 - Abs测试报告 - INFO - 用例【充值成功-整数】状态码断言成功
2023-05-12 14:50:29,826 - Abs测试报告 - ERROR - 用例【充值成功-整数】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
- {'code': '0', 'msg': 'OK'}
+ {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:50:29,828 - Abs测试报告 - DEBUG - 实际结果是{'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:50:29,828 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-12 14:50:30,062 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:50:30,062 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:50:30,133 - Abs测试报告 - INFO - 用例【充值成功-一位小数】状态码断言成功
2023-05-12 14:50:30,134 - Abs测试报告 - ERROR - 用例【充值成功-一位小数】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
- {'code': '0', 'msg': 'OK'}
+ {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:50:30,135 - Abs测试报告 - DEBUG - 实际结果是{'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:50:30,135 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-12 14:50:30,143 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:50:30,143 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:50:30,211 - Abs测试报告 - INFO - 用例【充值成功-两位小数】状态码断言成功
2023-05-12 14:50:30,212 - Abs测试报告 - ERROR - 用例【充值成功-两位小数】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
- {'code': '0', 'msg': 'OK'}
+ {'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:50:30,213 - Abs测试报告 - DEBUG - 实际结果是{'code': 1003, 'msg': '无法解析的JSON值，请检查token或Authorization请求头'}
2023-05-12 14:50:30,213 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-12 14:50:30,221 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-12 14:52:45,378 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-12 14:52:45,512 - Abs测试报告 - INFO - 注册成功
2023-05-12 14:52:45,512 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-12 14:52:45,619 - Abs测试报告 - INFO - 登录成功
2023-05-12 14:52:45,621 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:52:45,621 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:52:45,701 - Abs测试报告 - INFO - 用例【充值成功-整数】状态码断言成功
2023-05-12 14:52:45,702 - Abs测试报告 - ERROR - 用例【充值成功-整数】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 0, 'msg': 'OK'}
- {'code': '0', 'msg': 'OK'}
?          - -

+ {'code': 0, 'msg': 'OK'}
2023-05-12 14:52:45,704 - Abs测试报告 - DEBUG - 实际结果是{'code': 0, 'msg': 'OK'}
2023-05-12 14:52:45,704 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-12 14:52:45,942 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:52:45,942 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:52:46,030 - Abs测试报告 - INFO - 用例【充值成功-一位小数】状态码断言成功
2023-05-12 14:52:46,030 - Abs测试报告 - ERROR - 用例【充值成功-一位小数】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 0, 'msg': 'OK'}
- {'code': '0', 'msg': 'OK'}
?          - -

+ {'code': 0, 'msg': 'OK'}
2023-05-12 14:52:46,031 - Abs测试报告 - DEBUG - 实际结果是{'code': 0, 'msg': 'OK'}
2023-05-12 14:52:46,031 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-12 14:52:46,040 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:52:46,040 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:52:46,117 - Abs测试报告 - INFO - 用例【充值成功-两位小数】状态码断言成功
2023-05-12 14:52:46,118 - Abs测试报告 - ERROR - 用例【充值成功-两位小数】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 0, 'msg': 'OK'}
- {'code': '0', 'msg': 'OK'}
?          - -

+ {'code': 0, 'msg': 'OK'}
2023-05-12 14:52:46,119 - Abs测试报告 - DEBUG - 实际结果是{'code': 0, 'msg': 'OK'}
2023-05-12 14:52:46,119 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-12 14:52:46,127 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-12 14:54:09,893 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-12 14:54:10,028 - Abs测试报告 - INFO - 注册成功
2023-05-12 14:54:10,028 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-12 14:54:10,138 - Abs测试报告 - INFO - 登录成功
2023-05-12 14:54:10,139 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:54:10,139 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-12 14:54:10,232 - Abs测试报告 - INFO - 用例【充值成功-整数】状态码断言成功
2023-05-12 14:54:10,232 - Abs测试报告 - INFO - 用例【充值成功-整数】响应数据断言成功
2023-05-12 14:54:10,232 - Abs测试报告 - INFO - 用例【充值成功-整数】测试结束
2023-05-12 14:54:10,234 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:54:10,234 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-12 14:54:10,327 - Abs测试报告 - INFO - 用例【充值成功-一位小数】状态码断言成功
2023-05-12 14:54:10,327 - Abs测试报告 - INFO - 用例【充值成功-一位小数】响应数据断言成功
2023-05-12 14:54:10,327 - Abs测试报告 - INFO - 用例【充值成功-一位小数】测试结束
2023-05-12 14:54:10,329 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:54:10,329 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-12 14:54:10,401 - Abs测试报告 - INFO - 用例【充值成功-两位小数】状态码断言成功
2023-05-12 14:54:10,402 - Abs测试报告 - INFO - 用例【充值成功-两位小数】响应数据断言成功
2023-05-12 14:54:10,402 - Abs测试报告 - INFO - 用例【充值成功-两位小数】测试结束
2023-05-12 14:54:10,403 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-15 10:39:16,393 - Abs测试报告 - INFO - ==========start==========
2023-05-15 10:39:16,393 - Abs测试报告 - INFO - 用例加法开始测试
2023-05-15 10:39:16,393 - Abs测试报告 - INFO - 用例加法测试通过
2023-05-15 10:39:16,393 - Abs测试报告 - INFO - 用例减法开始测试
2023-05-15 10:39:16,393 - Abs测试报告 - INFO - 用例减法测试通过
2023-05-15 10:39:16,393 - Abs测试报告 - INFO - 用例乘法开始测试
2023-05-15 10:39:16,394 - Abs测试报告 - INFO - 用例乘法测试通过
2023-05-15 10:39:16,394 - Abs测试报告 - INFO - 用例除法开始测试
2023-05-15 10:39:16,394 - Abs测试报告 - INFO - 用例除法测试通过
2023-05-15 10:39:16,394 - Abs测试报告 - INFO - **********end**********
2023-05-15 10:39:16,394 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-15 10:39:16,569 - Abs测试报告 - INFO - 注册成功
2023-05-15 10:39:16,569 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-15 10:39:16,691 - Abs测试报告 - INFO - 登录成功
2023-05-15 10:39:16,692 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-15 10:39:16,692 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-15 10:39:16,780 - Abs测试报告 - INFO - 用例【充值成功-整数】状态码断言成功
2023-05-15 10:39:16,780 - Abs测试报告 - INFO - 用例【充值成功-整数】响应数据断言成功
2023-05-15 10:39:16,780 - Abs测试报告 - INFO - 用例【充值成功-整数】测试结束
2023-05-15 10:39:16,780 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-15 10:39:16,780 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-15 10:39:16,866 - Abs测试报告 - INFO - 用例【充值成功-一位小数】状态码断言成功
2023-05-15 10:39:16,866 - Abs测试报告 - INFO - 用例【充值成功-一位小数】响应数据断言成功
2023-05-15 10:39:16,866 - Abs测试报告 - INFO - 用例【充值成功-一位小数】测试结束
2023-05-15 10:39:16,866 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-15 10:39:16,867 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-15 10:39:16,953 - Abs测试报告 - INFO - 用例【充值成功-两位小数】状态码断言成功
2023-05-15 10:39:16,953 - Abs测试报告 - INFO - 用例【充值成功-两位小数】响应数据断言成功
2023-05-15 10:39:16,953 - Abs测试报告 - INFO - 用例【充值成功-两位小数】测试结束
2023-05-15 10:39:16,953 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-15 10:39:16,954 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-15 10:39:16,954 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-15 10:39:16,954 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-15 10:39:17,083 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】状态码断言成功
2023-05-15 10:39:17,084 - Abs测试报告 - ERROR - 用例【注册成功-不带昵称和类型】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 10:39:17,085 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 10:39:17,085 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 10:39:17,086 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-15 10:39:17,086 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-15 10:39:17,222 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】状态码断言成功
2023-05-15 10:39:17,222 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为0】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 10:39:17,223 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 10:39:17,223 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 10:39:17,223 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-15 10:39:17,224 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-15 10:39:17,357 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】状态码断言成功
2023-05-15 10:39:17,357 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为1】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 10:39:17,357 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 10:39:17,357 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 10:39:17,359 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-15 11:12:03,374 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 11:12:03,376 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 11:14:05,486 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 11:14:05,487 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 11:14:05,488 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 11:14:05,612 - Abs测试报告 - INFO - 用例【用户登录】状态码断言成功
2023-05-15 11:14:05,614 - Abs测试报告 - ERROR - 用例【用户登录】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 0, 'msg': 'OK'}
- {'code': '0', 'msg': 'OK'}
?          - -

+ {'code': 0, 'msg': 'OK'}
2023-05-15 11:14:05,616 - Abs测试报告 - DEBUG - 实际结果是{'code': 0, 'msg': 'OK'}
2023-05-15 11:14:05,616 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 11:14:05,853 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 11:14:40,076 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 11:14:40,077 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 11:14:40,077 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 11:14:40,215 - Abs测试报告 - INFO - 用例【用户登录】状态码断言成功
2023-05-15 11:14:40,215 - Abs测试报告 - INFO - 用例【用户登录】响应数据断言成功
2023-05-15 11:14:40,215 - Abs测试报告 - INFO - 用例【用户登录】测试结束
2023-05-15 11:14:40,216 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 11:15:13,728 - Abs测试报告 - INFO - ==========start==========
2023-05-15 11:15:13,728 - Abs测试报告 - INFO - 用例加法开始测试
2023-05-15 11:15:13,728 - Abs测试报告 - INFO - 用例加法测试通过
2023-05-15 11:15:13,728 - Abs测试报告 - INFO - 用例减法开始测试
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - 用例减法测试通过
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - 用例乘法开始测试
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - 用例乘法测试通过
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - 用例除法开始测试
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - 用例除法测试通过
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - **********end**********
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 11:15:13,729 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 11:15:13,885 - Abs测试报告 - INFO - 用例【用户登录】状态码断言成功
2023-05-15 11:15:13,885 - Abs测试报告 - INFO - 用例【用户登录】响应数据断言成功
2023-05-15 11:15:13,885 - Abs测试报告 - INFO - 用例【用户登录】测试结束
2023-05-15 11:15:13,885 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 11:15:13,885 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-15 11:15:16,125 - Abs测试报告 - INFO - 注册成功
2023-05-15 11:15:16,125 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-15 11:15:16,250 - Abs测试报告 - INFO - 登录成功
2023-05-15 11:15:16,250 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-15 11:15:16,250 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-15 11:15:16,364 - Abs测试报告 - INFO - 用例【充值成功-整数】状态码断言成功
2023-05-15 11:15:16,365 - Abs测试报告 - INFO - 用例【充值成功-整数】响应数据断言成功
2023-05-15 11:15:16,365 - Abs测试报告 - INFO - 用例【充值成功-整数】测试结束
2023-05-15 11:15:16,365 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-15 11:15:16,365 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-15 11:15:16,457 - Abs测试报告 - INFO - 用例【充值成功-一位小数】状态码断言成功
2023-05-15 11:15:16,457 - Abs测试报告 - INFO - 用例【充值成功-一位小数】响应数据断言成功
2023-05-15 11:15:16,457 - Abs测试报告 - INFO - 用例【充值成功-一位小数】测试结束
2023-05-15 11:15:16,457 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-15 11:15:16,457 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-15 11:15:16,547 - Abs测试报告 - INFO - 用例【充值成功-两位小数】状态码断言成功
2023-05-15 11:15:16,547 - Abs测试报告 - INFO - 用例【充值成功-两位小数】响应数据断言成功
2023-05-15 11:15:16,547 - Abs测试报告 - INFO - 用例【充值成功-两位小数】测试结束
2023-05-15 11:15:16,547 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-15 11:15:16,547 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-15 11:15:16,547 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-15 11:15:16,548 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-15 11:15:16,676 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】状态码断言成功
2023-05-15 11:15:16,677 - Abs测试报告 - ERROR - 用例【注册成功-不带昵称和类型】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 11:15:16,677 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 11:15:16,678 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 11:15:16,678 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-15 11:15:16,678 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-15 11:15:16,785 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】状态码断言成功
2023-05-15 11:15:16,785 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为0】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 11:15:16,785 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 11:15:16,785 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 11:15:16,786 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-15 11:15:16,786 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-15 11:15:16,896 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】状态码断言成功
2023-05-15 11:15:16,897 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为1】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 11:15:16,897 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 11:15:16,897 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 11:15:16,898 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-15 16:16:54,083 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 16:16:54,084 - Abs测试报告 - INFO - 用例【用户注册】开始测试
2023-05-15 16:16:54,084 - Abs测试报告 - INFO - 用例【用户注册】开始测试
2023-05-15 16:16:54,228 - Abs测试报告 - INFO - 用例【用户注册】状态码断言成功
2023-05-15 16:16:54,230 - Abs测试报告 - ERROR - 用例【用户注册】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\JetBrains\PyCharm 2022.2.3\plugins\python\helpers\pycharm\teamcity\diff_tools.py", line 33, in _patched_equals
    old(self, first, second, msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': 0, 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': 0, 'msg': 'OK'}
?          ^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 16:16:54,232 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 16:16:54,232 - Abs测试报告 - DEBUG - 预期结果是{'code': 0, 'msg': 'OK'}
2023-05-15 16:16:54,465 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 16:16:54,465 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 16:16:54,568 - Abs测试报告 - INFO - 用例【用户登录】状态码断言成功
2023-05-15 16:16:54,568 - Abs测试报告 - INFO - 用例【用户登录】响应数据断言成功
2023-05-15 16:16:54,568 - Abs测试报告 - INFO - 用例【用户登录】测试结束
2023-05-15 16:16:54,570 - Abs测试报告 - INFO - 用例【用户充值】开始测试
2023-05-15 16:16:54,570 - Abs测试报告 - INFO - 用例【用户充值】开始测试
2023-05-15 16:16:54,654 - Abs测试报告 - INFO - 用例【用户充值】状态码断言成功
2023-05-15 16:16:54,656 - Abs测试报告 - INFO - 用例【用户充值】响应数据断言成功
2023-05-15 16:16:54,656 - Abs测试报告 - INFO - 用例【用户充值】测试结束
2023-05-15 16:16:54,657 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 16:54:15,950 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 16:54:15,951 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-15 16:54:15,952 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-15 16:54:16,074 - Abs测试报告 - INFO - 用例【注册】状态码断言成功
2023-05-15 16:54:16,074 - Abs测试报告 - INFO - 用例【注册】响应数据断言成功
2023-05-15 16:54:16,074 - Abs测试报告 - INFO - 用例【注册】测试结束
2023-05-15 16:54:16,076 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-15 16:54:16,076 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-15 16:54:16,180 - Abs测试报告 - INFO - 用例【登录】状态码断言成功
2023-05-15 16:54:16,181 - Abs测试报告 - INFO - 用例【登录】响应数据断言成功
2023-05-15 16:54:16,181 - Abs测试报告 - INFO - 用例【登录】测试结束
2023-05-15 16:54:16,183 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-15 16:54:16,183 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-15 16:54:16,183 - Abs测试报告 - ERROR - 用例【充值】数据格式有误
2023-05-15 16:54:16,183 - Abs测试报告 - DEBUG - request:{
"headers": {"X-Lemonban-Media-Type": "lemonban.v2","Authorization":"Bearer #token#"},
"json":{"member_id":#member_id#,"amount":666}
}
2023-05-15 16:54:16,183 - Abs测试报告 - DEBUG - expect:{
"code": 0, "msg": "OK"
}
2023-05-15 16:54:16,427 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - ==========start==========
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例加法开始测试
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例加法测试通过
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例减法开始测试
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例减法测试通过
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例乘法开始测试
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例乘法测试通过
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例除法开始测试
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - 用例除法测试通过
2023-05-15 16:59:10,912 - Abs测试报告 - INFO - **********end**********
2023-05-15 16:59:10,913 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 16:59:10,913 - Abs测试报告 - INFO - 用例【用户注册】开始测试
2023-05-15 16:59:10,913 - Abs测试报告 - INFO - 用例【用户注册】开始测试
2023-05-15 16:59:11,028 - Abs测试报告 - INFO - 用例【用户注册】状态码断言成功
2023-05-15 16:59:11,028 - Abs测试报告 - ERROR - 用例【用户注册】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': 0, 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': 0, 'msg': 'OK'}
?          ^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 16:59:11,029 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 16:59:11,029 - Abs测试报告 - DEBUG - 预期结果是{'code': 0, 'msg': 'OK'}
2023-05-15 16:59:11,029 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 16:59:11,029 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-15 16:59:11,147 - Abs测试报告 - INFO - 用例【用户登录】状态码断言成功
2023-05-15 16:59:11,147 - Abs测试报告 - INFO - 用例【用户登录】响应数据断言成功
2023-05-15 16:59:11,147 - Abs测试报告 - INFO - 用例【用户登录】测试结束
2023-05-15 16:59:11,147 - Abs测试报告 - INFO - 用例【用户充值】开始测试
2023-05-15 16:59:11,147 - Abs测试报告 - INFO - 用例【用户充值】开始测试
2023-05-15 16:59:11,236 - Abs测试报告 - INFO - 用例【用户充值】状态码断言成功
2023-05-15 16:59:11,236 - Abs测试报告 - INFO - 用例【用户充值】响应数据断言成功
2023-05-15 16:59:11,236 - Abs测试报告 - INFO - 用例【用户充值】测试结束
2023-05-15 16:59:11,236 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 16:59:11,236 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 16:59:11,236 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-15 16:59:11,237 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-15 16:59:11,339 - Abs测试报告 - INFO - 用例【注册】状态码断言成功
2023-05-15 16:59:11,339 - Abs测试报告 - INFO - 用例【注册】响应数据断言成功
2023-05-15 16:59:11,339 - Abs测试报告 - INFO - 用例【注册】测试结束
2023-05-15 16:59:11,340 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-15 16:59:11,340 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-15 16:59:11,442 - Abs测试报告 - INFO - 用例【登录】状态码断言成功
2023-05-15 16:59:11,442 - Abs测试报告 - INFO - 用例【登录】响应数据断言成功
2023-05-15 16:59:11,442 - Abs测试报告 - INFO - 用例【登录】测试结束
2023-05-15 16:59:11,442 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-15 16:59:11,442 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-15 16:59:11,442 - Abs测试报告 - ERROR - 用例【充值】数据格式有误
2023-05-15 16:59:11,442 - Abs测试报告 - DEBUG - request:{
"headers": {"X-Lemonban-Media-Type": "lemonban.v2","Authorization":"Bearer #token#"},
"json":{"member_id":#member_id#,"amount":666}
}
2023-05-15 16:59:11,443 - Abs测试报告 - DEBUG - expect:{
"code": 0, "msg": "OK"
}
2023-05-15 16:59:11,445 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-15 16:59:11,445 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-15 16:59:11,555 - Abs测试报告 - INFO - 注册成功
2023-05-15 16:59:11,555 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-15 16:59:11,664 - Abs测试报告 - INFO - 登录成功
2023-05-15 16:59:11,665 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-15 16:59:11,665 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-15 16:59:11,754 - Abs测试报告 - INFO - 用例【充值成功-整数】状态码断言成功
2023-05-15 16:59:11,754 - Abs测试报告 - INFO - 用例【充值成功-整数】响应数据断言成功
2023-05-15 16:59:11,754 - Abs测试报告 - INFO - 用例【充值成功-整数】测试结束
2023-05-15 16:59:11,754 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-15 16:59:11,755 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-15 16:59:11,845 - Abs测试报告 - INFO - 用例【充值成功-一位小数】状态码断言成功
2023-05-15 16:59:11,845 - Abs测试报告 - INFO - 用例【充值成功-一位小数】响应数据断言成功
2023-05-15 16:59:11,845 - Abs测试报告 - INFO - 用例【充值成功-一位小数】测试结束
2023-05-15 16:59:11,846 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-15 16:59:11,846 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-15 16:59:11,935 - Abs测试报告 - INFO - 用例【充值成功-两位小数】状态码断言成功
2023-05-15 16:59:11,935 - Abs测试报告 - INFO - 用例【充值成功-两位小数】响应数据断言成功
2023-05-15 16:59:11,935 - Abs测试报告 - INFO - 用例【充值成功-两位小数】测试结束
2023-05-15 16:59:11,935 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-15 16:59:11,935 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-15 16:59:11,936 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-15 16:59:11,936 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-15 16:59:12,038 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】状态码断言成功
2023-05-15 16:59:12,039 - Abs测试报告 - ERROR - 用例【注册成功-不带昵称和类型】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 16:59:12,039 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 16:59:12,039 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 16:59:12,040 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-15 16:59:12,040 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-15 16:59:12,141 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】状态码断言成功
2023-05-15 16:59:12,141 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为0】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 16:59:12,142 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 16:59:12,142 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 16:59:12,142 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-15 16:59:12,142 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-15 16:59:12,239 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】状态码断言成功
2023-05-15 16:59:12,239 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为1】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 103, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-15 16:59:12,239 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-15 16:59:12,239 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-15 16:59:12,240 - Abs测试报告 - INFO - **********注册接口测试结束**********
2023-05-15 17:03:54,156 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-15 17:03:54,158 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-15 17:03:54,158 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-15 17:03:54,286 - Abs测试报告 - INFO - 用例【注册】状态码断言成功
2023-05-15 17:03:54,286 - Abs测试报告 - INFO - 用例【注册】响应数据断言成功
2023-05-15 17:03:54,286 - Abs测试报告 - INFO - 用例【注册】测试结束
2023-05-15 17:03:54,288 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-15 17:03:54,288 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-15 17:03:54,397 - Abs测试报告 - INFO - 用例【登录】状态码断言成功
2023-05-15 17:03:54,398 - Abs测试报告 - INFO - 用例【登录】响应数据断言成功
2023-05-15 17:03:54,398 - Abs测试报告 - INFO - 用例【登录】测试结束
2023-05-15 17:03:54,400 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-15 17:03:54,400 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-15 17:03:54,479 - Abs测试报告 - INFO - 用例【充值】状态码断言成功
2023-05-15 17:03:54,479 - Abs测试报告 - INFO - 用例【充值】响应数据断言成功
2023-05-15 17:03:54,480 - Abs测试报告 - INFO - 用例【充值】测试结束
2023-05-15 17:03:54,481 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-16 11:33:25,474 - Abs测试报告 - INFO - ==========start==========
2023-05-16 11:33:25,474 - Abs测试报告 - INFO - 用例加法开始测试
2023-05-16 11:33:25,474 - Abs测试报告 - INFO - 用例加法测试通过
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例减法开始测试
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例减法测试通过
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例乘法开始测试
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例乘法测试通过
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例除法开始测试
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例除法测试通过
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - **********end**********
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例【用户注册】开始测试
2023-05-16 11:33:25,475 - Abs测试报告 - INFO - 用例【用户注册】开始测试
2023-05-16 11:33:25,617 - Abs测试报告 - INFO - 用例【用户注册】状态码断言成功
2023-05-16 11:33:25,617 - Abs测试报告 - ERROR - 用例【用户注册】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 108, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': 0, 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': 0, 'msg': 'OK'}
?          ^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-16 11:33:25,618 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-16 11:33:25,618 - Abs测试报告 - DEBUG - 预期结果是{'code': 0, 'msg': 'OK'}
2023-05-16 11:33:25,618 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-16 11:33:25,618 - Abs测试报告 - INFO - 用例【用户登录】开始测试
2023-05-16 11:33:25,686 - Abs测试报告 - INFO - 用例【用户登录】状态码断言成功
2023-05-16 11:33:25,686 - Abs测试报告 - INFO - 用例【用户登录】响应数据断言成功
2023-05-16 11:33:25,686 - Abs测试报告 - INFO - 用例【用户登录】测试结束
2023-05-16 11:33:25,686 - Abs测试报告 - INFO - 用例【用户充值】开始测试
2023-05-16 11:33:25,686 - Abs测试报告 - INFO - 用例【用户充值】开始测试
2023-05-16 11:33:25,728 - Abs测试报告 - INFO - 用例【用户充值】状态码断言成功
2023-05-16 11:33:25,728 - Abs测试报告 - INFO - 用例【用户充值】响应数据断言成功
2023-05-16 11:33:25,728 - Abs测试报告 - INFO - 用例【用户充值】测试结束
2023-05-16 11:33:25,728 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-16 11:33:25,728 - Abs测试报告 - INFO - ----------贷款主流程接口测试开始----------
2023-05-16 11:33:25,729 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-16 11:33:25,729 - Abs测试报告 - INFO - 用例【注册】开始测试
2023-05-16 11:33:25,798 - Abs测试报告 - INFO - 用例【注册】状态码断言成功
2023-05-16 11:33:25,798 - Abs测试报告 - INFO - 用例【注册】响应数据断言成功
2023-05-16 11:33:25,798 - Abs测试报告 - INFO - 用例【注册】测试结束
2023-05-16 11:33:25,798 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-16 11:33:25,798 - Abs测试报告 - INFO - 用例【登录】开始测试
2023-05-16 11:33:25,865 - Abs测试报告 - INFO - 用例【登录】状态码断言成功
2023-05-16 11:33:25,866 - Abs测试报告 - INFO - 用例【登录】响应数据断言成功
2023-05-16 11:33:25,866 - Abs测试报告 - INFO - 用例【登录】测试结束
2023-05-16 11:33:25,867 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-16 11:33:25,867 - Abs测试报告 - INFO - 用例【充值】开始测试
2023-05-16 11:33:25,918 - Abs测试报告 - INFO - 用例【充值】状态码断言成功
2023-05-16 11:33:25,918 - Abs测试报告 - INFO - 用例【充值】响应数据断言成功
2023-05-16 11:33:25,918 - Abs测试报告 - INFO - 用例【充值】测试结束
2023-05-16 11:33:25,918 - Abs测试报告 - INFO - **********贷款主流程接口测试结束**********
2023-05-16 11:33:25,918 - Abs测试报告 - INFO - ----------充值接口测试开始----------
2023-05-16 11:33:26,024 - Abs测试报告 - INFO - 注册成功
2023-05-16 11:33:26,024 - Abs测试报告 - INFO - 注册用户15601665487成功
2023-05-16 11:33:26,125 - Abs测试报告 - INFO - 登录成功
2023-05-16 11:33:26,126 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-16 11:33:26,126 - Abs测试报告 - INFO - 用例【充值成功-整数】开始测试
2023-05-16 11:33:26,169 - Abs测试报告 - INFO - 用例【充值成功-整数】状态码断言成功
2023-05-16 11:33:26,169 - Abs测试报告 - INFO - 用例【充值成功-整数】响应数据断言成功
2023-05-16 11:33:26,169 - Abs测试报告 - INFO - 用例【充值成功-整数】测试结束
2023-05-16 11:33:26,169 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-16 11:33:26,169 - Abs测试报告 - INFO - 用例【充值成功-一位小数】开始测试
2023-05-16 11:33:26,215 - Abs测试报告 - INFO - 用例【充值成功-一位小数】状态码断言成功
2023-05-16 11:33:26,215 - Abs测试报告 - INFO - 用例【充值成功-一位小数】响应数据断言成功
2023-05-16 11:33:26,215 - Abs测试报告 - INFO - 用例【充值成功-一位小数】测试结束
2023-05-16 11:33:26,215 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-16 11:33:26,216 - Abs测试报告 - INFO - 用例【充值成功-两位小数】开始测试
2023-05-16 11:33:26,260 - Abs测试报告 - INFO - 用例【充值成功-两位小数】状态码断言成功
2023-05-16 11:33:26,261 - Abs测试报告 - INFO - 用例【充值成功-两位小数】响应数据断言成功
2023-05-16 11:33:26,261 - Abs测试报告 - INFO - 用例【充值成功-两位小数】测试结束
2023-05-16 11:33:26,261 - Abs测试报告 - INFO - **********充值接口测试结束**********
2023-05-16 11:33:26,261 - Abs测试报告 - INFO - ----------注册接口测试开始----------
2023-05-16 11:33:26,261 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-16 11:33:26,261 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】开始测试
2023-05-16 11:33:26,331 - Abs测试报告 - INFO - 用例【注册成功-不带昵称和类型】状态码断言成功
2023-05-16 11:33:26,331 - Abs测试报告 - ERROR - 用例【注册成功-不带昵称和类型】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 108, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-16 11:33:26,332 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-16 11:33:26,332 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-16 11:33:26,333 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-16 11:33:26,333 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】开始测试
2023-05-16 11:33:26,404 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为0】状态码断言成功
2023-05-16 11:33:26,404 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为0】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 108, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-16 11:33:26,405 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-16 11:33:26,405 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-16 11:33:26,405 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-16 11:33:26,406 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】开始测试
2023-05-16 11:33:26,476 - Abs测试报告 - INFO - 用例【注册成功-带昵称-类型为1】状态码断言成功
2023-05-16 11:33:26,477 - Abs测试报告 - ERROR - 用例【注册成功-带昵称-类型为1】响应数据断言失败
Traceback (most recent call last):
  File "D:\learnpython\autoTest\testcases\base_cases.py", line 108, in assert_json_response
    self.assertEqual(self.case['expect'], res)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Program Files\Python310\lib\unittest\case.py", line 1144, in assertDictEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Program Files\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: {'code': '0', 'msg': 'OK'} != {'code': 2, 'msg': '账号已存在'}
- {'code': '0', 'msg': 'OK'}
?          ^^^          ^^

+ {'code': 2, 'msg': '账号已存在'}
?          ^          ^^^^^

2023-05-16 11:33:26,477 - Abs测试报告 - DEBUG - 实际结果是{'code': 2, 'msg': '账号已存在'}
2023-05-16 11:33:26,477 - Abs测试报告 - DEBUG - 预期结果是{'code': '0', 'msg': 'OK'}
2023-05-16 11:33:26,478 - Abs测试报告 - INFO - **********注册接口测试结束**********
